src/conn/conn_log.c

```diff
@@ -314,6 +314,8 @@ __wt_log_truncate_files(
 
 	WT_UNUSED(cfg);
 	conn = S2C(session);
+	if (!FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED))
+		return (0);
```

Comment:
IMHO, I'd suggest re-ordering these lines and tossing in some whitespace, the initializaiton of "log" in-between two tests of connection flags seems messy:
```
WT_UNUSED(cfg);

conn = S2C(session);
if (!FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED))
        return (0);
if (F_ISSET(conn, WT_CONN_SERVER_RUN) && 
    FLD_ISSET(conn->log_flags, WT_CONN_LOG_ARCHIVE))
        WT_RET_MSG(session, EINVAL,
            "Attempt to archive manually while a server is running");

log = conn->log;
```